spec/fixtures/json/expected_output.json

```diff
@@ -0,0 +1,51 @@
+{
```

Comment:
This file was for my notes on what sort of data should be returned by the Serializer.